src/pageComponents/search/Results.jsx

```diff
@@ -59,6 +60,15 @@ MediaWrapper.propTypes = {
   url: PropTypes.string.isRequired
 };
 
+function formatGuide(text) {
```

Comment:
Is this the right place for this function?

Reply:
I think it might be better to move this to the `searchEpic`, which mean is will only be done once for each result payload, instead of every render

Reply:
What I was doing before was a bit naughty to be honest